diff --git a/node_modules/@types/react/index.d.ts b/node_modules/@types/react/index.d.ts
index 6ea73ef..a2b1021 100644
--- a/node_modules/@types/react/index.d.ts
+++ b/node_modules/@types/react/index.d.ts
@@ -1012,12 +1012,12 @@ declare namespace React {
          */
         context: unknown;
 
-        constructor(props: P);
+        constructor(props: P & { children?: ReactNode });
         /**
          * @deprecated
          * @see {@link https://legacy.reactjs.org/docs/legacy-context.html React Docs}
          */
-        constructor(props: P, context: any);
+        constructor(props: P & { children?: ReactNode }, context: any);
 
         // We MUST keep setState() as a unified signature because it allows proper checking of the method return type.
         // See: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18365#issuecomment-351013257
@@ -1125,7 +1125,7 @@ declare namespace React {
      */
     interface FunctionComponent<P = {}> {
         (
-            props: P,
+            props: P & { children?: ReactNode },
             /**
              * @deprecated
              *
