diff --git a/node_modules/@langchain/anthropic/dist/chat_models.cjs b/node_modules/@langchain/anthropic/dist/chat_models.cjs
--- a/node_modules/@langchain/anthropic/dist/chat_models.cjs
+++ b/node_modules/@langchain/anthropic/dist/chat_models.cjs
@@ -615,8 +615,9 @@ class ChatAnthropic extends core_1.BaseChatModel {
         this.modelName = fields?.model ?? fields?.modelName ?? this.model;
         this.model = this.modelName;
         this.invocationKwargs = fields?.invocationKwargs ?? {};
         if (this.model.includes("opus-4-1") ||
+            this.model.includes("opus-4-5") ||
             this.model.includes("sonnet-4-5") ||
             this.model.includes("haiku-4-5")) {
             this.temperature =
                 fields?.temperature === null ? undefined : fields?.temperature;
             this.topP = fields?.topP === null ? undefined : fields?.topP;
        }


diff --git a/node_modules/@langchain/anthropic/dist/chat_models.js b/node_modules/@langchain/anthropic/dist/chat_models.js
--- a/node_modules/@langchain/anthropic/dist/chat_models.js
+++ b/node_modules/@langchain/anthropic/dist/chat_models.js
@@ -612,8 +612,9 @@ export class ChatAnthropic extends BaseChatModel {
         this.modelName = fields?.model ?? fields?.modelName ?? this.model;
         this.model = this.modelName;
         this.invocationKwargs = fields?.invocationKwargs ?? {};
         if (this.model.includes("opus-4-1") ||
+            this.model.includes("opus-4-5") ||
             this.model.includes("sonnet-4-5") ||
             this.model.includes("haiku-4-5")) {
             this.temperature =
                 fields?.temperature === null ? undefined : fields?.temperature;
             this.topP = fields?.topP === null ? undefined : fields?.topP;
         }
